.github/workflows/verify.yml

```diff
@@ -42,4 +42,65 @@ jobs:
         run: npm run test:unit
 
       - name: Test e2e
-        run: npm run test:e2e:ci
\ No newline at end of file
+        run: npm run test:e2e:ci
+
+      - name: Archive frontend build output
+        uses: actions/upload-artifact@v3
+        with:
+          name: frontend-build
+          path: packages/frontend/dist
+          retention-days: 1
+
+      - name: Archive backend build artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: backend-build
+          path: packages/backend/dist
+          retention-days: 1
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: [verify]
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download frontend build artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: frontend-build
+          path: packages/frontend/dist
+
+      - name: Download backend build artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: backend-build
+          path: packages/backend/dist
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Copy package-lock from root
+        run: cp package-lock.json packages/backend
+
+      - name: Copy node_modules from root
+        run: cp -R node_modules packages/backend
+
+      - name: Remove @dependency-health-monitor from node_modules in backend
+        run: cd packages/backend && rm -rf node_modules/@dependency-health-monitor
+
+      - name: Install SWA
```

Comment:
It's not considered best practice to install globally on CI/CD. This is why the `npx` command was invented. See next comment.

Reply:
Resolved in fcb783d9b104aaaa75a840e823d2e1188be11108